starters/vue3-xstate-css/package.json

```diff
@@ -11,28 +11,29 @@
     "test:unit:dev": "cypress open --component",
     "build-only": "vite build",
     "type-check": "vue-tsc --noEmit -p tsconfig.cypress-ct.json --composite false",
-    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore"
+    "lint": "eslint . --ext .vue,.js,.jsx,.ts,.tsx, --fix --ignore-path .gitignore",
+    "format": "prettier --write ./src --ignore-path .gitignore"
```

Comment:
You've introduced the `.prettierignore` file but are not using it here. Was this done deliberately? And if so, can you tell me why we have the `.prettierignore` file?

Reply:
....this is an excellent point. lol In all honesty, I was mostly just copying the other kits. But this is also exactly why it would've been made! I'm going to compare the two files and either remove the prettier ignore or replace the git ignore in the command. 👍🏼